src/groups/mqb/mqbmock/mqbmock_cluster.cpp

```diff
@@ -231,7 +235,12 @@ Cluster::Cluster(bdlbb::BlobBufferFactory* bufferFactory,
 , d_processor()
 {
     // PRECONDITIONS
-    BSLS_ASSERT_OPT(isClusterMember || !isLeader);
+    if (isClusterMember) {
+        BSLS_ASSERT_OPT(!clusterNodeDefs.empty());
```

Comment:
Early safety check. Providing an empty `clusterNodeDefs` make it certain that `selfNode` will be nullptr.